versions.gradle

```diff
@@ -25,13 +25,13 @@ def versions = [:]
 // GeckoView versions can be found here:
 // https://maven.mozilla.org/?prefix=maven2/org/mozilla/geckoview/
 versions.gecko_view = "77.0.20200422093542"
-versions.android_components = "28.0.1"
+versions.android_components = "37.0.0"
```

Comment:
This could be bumped to `39.0.1` or `40.0.0` - and appservices to `0.58.2`.

Reply:
Updated